_people/jennifer-aube.md

```diff
@@ -6,13 +6,15 @@ name: Jennifer Aube
 image: jennifer-aube.jpg
 categories:
  - 
+sidenav: docs
+permalink: /about/people/jennifer-aube
```

Comment:
@jaube05 this change changes the URL for your profile vs the others. Noticed in the pa11y-ci run:

```
 > https://accessibility.civicactions.com/about/people/jennifer-aube - 0 errors
 > https://accessibility.civicactions.com/people/daniel-mundra - 0 errors
```